packages/pn-personagiuridica-webapp/src/navigation/routes.tsx

```diff
@@ -146,6 +147,36 @@ function Router() {
                     </PrivateRoute>
                   }
                 />
+                <Route
+                  path={routes.REGISTRA_CHIAVE_PUBBLICA}
+                  element={
+                    <PrivateRoute
+                      currentRoles={[]}
+                      requiredRoles={[]}
+                      additionalCondition={IS_B2B_ENABLED}
+                      redirectTo={<NotFound />}
+                    >
+                      <NewPublicKey />
+                    </PrivateRoute>
+                  }
+                />
+
+                {/* <Route
```

Comment:
this can be removed?

Reply:
This was the code I created while trying to make the path param work for public key rotation.
Since it was not working at all, even though I was following what I found inside the official react router guide, I decided to move on and pass the parameter through the query string as done until now.
I thought to comment it in case we'd like to give it a second chance.

Reply:
1. Remove the commented code.
2. As written here https://stackoverflow.com/questions/70005601/alternate-way-for-optional-parameters-in-v6, our version of react router doesn't support optional parameters. 
3. Follow this guide https://www.cluemediator.com/url-parameters-with-react-router-v6

Reply:
@mflauti point 6 hasn't been done